addons/l10n_ua/data/template/account.tax-ua_psbo.csv

```diff
@@ -1,79 +1,79 @@
 "id","sequence","name","description","invoice_label","amount","type_tax_use","tax_group_id","price_include","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id"
-"sale_tax_template_vat20_psbo","9","Реалізація ПДВ 20%","","+ ПДВ 20%","20.0","sale","tax_group_vat20","","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_psbp_6431"
+"sale_tax_template_vat20_psbo","9","ПДВ 20%","","+ ПДВ 20%","20.0","sale","tax_group_vat20","","base","invoice",""

```

Comment:
Does this mean, the 6431 account should rather be used on the tax group as tax payable account? For the tax closing entry to work out the box in Ukraine?